### ISSUE TITLE
[A11y] iOS should not attempt to focus hidden elements if the parent does not have implicit scroll flag

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#88587]
Framework load a few more off screen element in a scrollable to achieve scrolling in iOS, those elements are marked hidden but still a11y focusable. We should make it so that VoiceOver can only discover these nodes when the parent has implicit scroll. This can enable uses of cache extend to preload element in page view

Some ideas:
1. Make hidden element unfocusable if it is not a direct child of a implicit scroll node, this can be done in isAccessibilityElement getter

2. Update pageview to have a mechanism to load the next page without attribute to semantics

**Comment by chunhtai** [comment]
wrong issue
